api/src/test/java/org/openmrs/ui/framework/converter/StringToGlobalPropertyConverterTest.java

```diff
@@ -11,7 +11,7 @@ public class StringToGlobalPropertyConverterTest extends BaseModuleContextSensit
 	public void convert_shouldConvertStringToGlobalProperty() {
 		StringToGlobalPropertyConverter converter = new StringToGlobalPropertyConverter();
 		GlobalProperty prop = converter.convert("locale.allowed.list");
-		Assert.assertEquals("en", prop.getPropertyValue());
+		Assert.assertEquals("en_GB", prop.getPropertyValue());
```

Comment:
Did you change this accidentally?

Reply:
After updating to platform 2.0.0  this test failed with the following error:

```
junit.framework.ComparisonFailure: expected:<en[]> but was:<en[_GB]>
	at org.openmrs.ui.framework.converter.StringToGlobalPropertyConverterTest.convert_shouldConvertStringToGlobalProperty
```

That's why I updated the test.

Reply:
OK. That makes sense. 😊